install-and-configure/install/environment.md

```diff
@@ -5,7 +5,7 @@ Kubecost requires a Kubernetes cluster to be deployed.
 ## Supported Kubernetes versions
 
 * Kubecost runs in any Kubernetes v1.8 (Aug 2017) or greater environment.
```

Comment:
This statement is not true currently.

Reply:
Can you elaborate?

Reply:
@chipzoller  Can you fill in the detail here? Would be great to get this updated.